src/js/components/Select/stories/Basics.js

```diff
@@ -84,4 +94,5 @@ for (let i = 1; i <= 200; i += 1) {
 
 storiesOf('Select', module)
   .add('Simple', () => <SimpleSelect />)
-  .add('Custom Theme', () => <SimpleSelect open theme={customRoundedTheme} />);
+  .add('Custom Theme', () => <SimpleSelect open theme={customRoundedTheme} />)
+  .add('Custom Theme Icons', () => <SimpleSelect theme={iconTheme} />);
```

Comment:
I think we have enough stories for Select, so we can just put this on a Custom Theme example rather than creating a new story.

Reply:
Sounds good. I'll update that story.